ci/do_ci.sh

```diff
@@ -112,33 +112,40 @@ EOF
   examples/plugin/load/load_plugin_example ${PLUGIN_DIR}/libexample_plugin.so /dev/null
   exit 0
 elif [[ "$1" == "bazel.test" ]]; then
+  bazel clean --expunge
```

Comment:
I am struggling with Bazel build failing on GitHub action. But working totally fine on my local setup. It fails around `opentelemetry-protobuf` repo, not finding file..